src/igvfd/schemas/construct_library_set.json

```diff
@@ -709,6 +709,16 @@
                 "alleles",
                 "targeton"
             ],
+            "enum_descriptions": {
+                "tile": "The construct library was designed to target a tile of a protein region.",
```

Comment:
"the construct library was designed to tile over a region of the genome or protein"

Reply:
fixed